deploy.sh

```diff
@@ -9,6 +9,8 @@ source "$(dirname "${0}")/build.env"
 
 BRANCH_NAME=${BRANCH_NAME:-""}
 GITHUB_TOKEN=${GITHUB_TOKEN:-""}
+GITHUB_USER=${GITHUB_USER:-"ceph-csi-bot"}
+GITHUB_EMAIL=${GITHUB_EMAIL:-"ceph-csi-bot@ceph.io"}
```

Comment:
If these variables are set for the GitHub action, set them to something else here (`autobuild-bot` and `noreply@example.com`).

For the GitHub variable, you can use `ceph-csi-bot@users.noreply.github.com`.

Reply:
done